client/signup/steps/website-content/page-details.tsx

```diff
@@ -22,7 +22,7 @@ export function PageDetails( {
 	onChangeField,
 }: {
 	page: PageData;
-	formErrors: any;
+	formErrors: Record< string, ReactChild >;
```

Comment:
This causes a TS error in https://github.com/Automattic/wp-calypso/blob/4dd521885f9d795412714a6ece6f4315d6dbb7b5/client/signup/steps/website-content/section-generator.tsx#L25
```
Type 'ValidationErrors' is not assignable to type 'Record<string, ReactChild>'.
  'string' index signatures are incompatible.
    Type 'ReactChild | null' is not assignable to type 'ReactChild'.
      Type 'null' is not assignable to type 'ReactChild'.
```